tools/kotlin-native-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/experimental/internal/KotlinNativeMainComponent.kt

```diff
@@ -82,5 +82,8 @@ open class KotlinNativeMainComponent @Inject constructor(
 
         @JvmStatic
         val KLIBRARY = OutputKind.KLIBRARY
+
+        @JvmStatic
+        val FRAMEWORK = OutputKind.FRAMEWORK
```

Comment:
What about static and dynamic libs as well?

Reply:
Adding them is not very complicated but I think that it's better to focus on integration with Kotlin MPP at the moment.